conbench/entities/run.py

```diff
@@ -59,12 +63,24 @@ def create(data):
 
         # create if not exists
         commit = Commit.first(sha=sha, repository=repository)
+
         if not commit:
-            github = get_github_commit(
-                repository=repository, pr_number=pr_number, branch=branch, sha=sha
-            )
-            if github:
-                commit = Commit.create_github_context(sha, repository, github)
+
+            # Try to fetch data via GitHub HTTP API
+            gh_commit_dict = None
+            try:
+                # get_github_commit() may raise all those exceptions that can
+                # happen during an HTTP request cycle.
+                gh_commit_dict = get_github_commit(
+                    repository=repository, pr_number=pr_number, branch=branch, sha=sha
+                )
+            except Exception as exc:
+                log.info(
+                    "treat as unknown commit: error during get_github_commit(): %s", exc
```

Comment:
Austin, is this "treat as unknown commit" meaningful/correct?

Reply:
Yes, that should be the case here!